helm_deploy/prisoner-content-hub-backend/values.yaml

```diff
@@ -75,7 +75,7 @@ ingress:
     nginx.ingress.kubernetes.io/proxy-body-size: 500m
 
 cron:
-  schedule: "*/* 3 * * *"
+  schedule: "0 * * * *"
```

Comment:
Do we want to match what is currently scheduled in Drupal? `0 0/3 * * *` should match IIRC

Reply:
Just saw the PR comments... never-mind, makes sense

Reply:
I couldn't recall the rationale around the current schedule?

I can change to 0 */3 * * *

Reply:
Unless there's a reason not to I think we should make it fairly frequent. The job should complete in seconds (it's a tiny database) so it just means the content will be indexed for search sooner, right?

Reply:
Makes sense to me, keep it frequent, less and often 👍